examples/examples.md

```diff
@@ -1,6 +1,6 @@
 # Basecoin Example
 
-Here we explain how to get started with a basic Basecoin blockchain, how
+Here we explain how to get started with a simple Basecoin blockchain, how
```

Comment:
i try to avoid using the word "simple" in docs. read a study once upon a time that showed noobs get turned off from a project really easily if it's described as simple and they stumble at all. basic is fine :)